pom.xml

```diff
@@ -29,13 +29,14 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <maven.build.timestamp.format>yyyy-MM-dd-HH-mm</maven.build.timestamp.format>
+    <gcloud.api.version>0.13.0-beta</gcloud.api.version>
```

Comment:
@meltsufin I can't see a 0.17.1 in maven central.  But I can see a 1.0.0! http://repo2.maven.org/maven2/com/google/cloud/google-cloud-logging/